### DISCUSSION TITLE
如何默认就是顶部菜单，不需要左侧菜单，也不需要用设置去设置导航模式

### DISCUSSION THREAD

**Discussion by sunfeng90** [vbenjs/vue-vben-admin#3115] [General]
只需要顶部菜单模式，默认就是。不需要动态设置。有没有配置的地方

**Comment by DesignHhuang** [comment]
![image](https://github.com/vbenjs/vue-vben-admin/assets/3368397/8bae115c-7de3-4d83-a8a3-720adddd7a91)
在projectSetting中，设置后清缓存重启就可以了
